.gitignore

```diff
@@ -408,3 +408,8 @@ erasure/src/kernames.ml
 erasure/src/kernames.mli
 erasure/src/reflect.ml
 erasure/src/reflect.mli
+erasure/src/pCUICWfUniverses.ml
+erasure/src/pCUICWfUniverses.mli
+erasure/src/signature.ml
+erasure/src/signature.mli
+erasure/src/eOptimizePropDiscr.ml
```

Comment:
There is also an .mli file that would be nice to ignore/remove (although it doesn't have to be here :) )

Reply:
Indeed